src/bundle/Resources/views/themes/admin/ui/menu/main_base.html.twig

```diff
@@ -64,11 +82,20 @@
             <li class="ibexa-main-menu__separator ibexa-main-menu__separator--top"></li>
         {%- endif -%}
 
-        <li{{ knp_menu.attributes(attributes) }}>
+        <li {{ knp_menu.attributes(attributes) }}>
             {%- if item.uri is not empty and (not matcher.isCurrent(item) or options.currentAsLink) %}
                 {{ block('linkElement') }}
             {%- else %}
                 {{ block('spanElement') }}
+
+                {% if item.children and secondLevel is defined and secondLevel %}
```

Comment:
It's possible that depth is available here, and could be used instead of `secondLevel` variable. It might suit it better?